templates/eap-cd-sso-s2i.json

```diff
@@ -136,6 +136,13 @@
             "value": "openshift",
             "required": true
         },
+        {
+            "displayName": "JGroups Encryption Protocol",
+            "description": "Protocol to use for JGroups (SYM_ENCRYPT, ASYM_ENCRYPT)"
+            "name": "JGROUPS_ENCRYPT_PROTOCOL",
+            "value": "SYM_ENCRYPT",
```

Comment:
Like suggested in the comment. I would change this to `ASYM_ENCRYPT`, since `SYM_ENCRYPT` is still the default in the EAP 7.1 image here:
  https://github.com/jboss-openshift/cct_module/blob/master/os-eap-launch/added/launch/jgroups.sh#L24
or for EAP CD images here:
  https://github.com/jboss-container-images/jboss-eap-modules/blob/master/jboss-eap-config-jgroups/added/launch/jgroups.sh#L103.

And remove all the variables required by `SYM_ENCRYPT` protocol:
- `JGROUPS_ENCRYPT_SECRET`
- `JGROUPS_ENCRYPT_NAME`
- `JGROUPS_ENCRYPT_PASSWORD`
- `JGROUPS_ENCRYPT_KEYSTORE_DIR`
- `JGROUPS_ENCRYPT_KEYSTORE`

since they can still be used by template instantiation via `-e parameter_name=paremeter_value` constructs (effectively making both of the protocols possible / configurable)